core/sparqlbuilder/pom.xml

```diff
@@ -15,5 +15,10 @@
 			<artifactId>rdf4j-model</artifactId>
 			<version>${project.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>org.eclipse.rdf4j</groupId>
+			<artifactId>rdf4j-rio-turtle</artifactId>
```

Comment:
I would prefer if we didn't need this, especially since it's just for the use of a single utility class. Let's have a think about maybe moving or copying the code to a better place.

Reply:
Sure. Somewhere in core?

Reply:
Yep. `rdf4j-model` would be a good place. It already has a class `URIUtil` which has several closely related utility methods - I wonder if we could stick this in there somewhere. Bonus: it's already a dependency for the sparqlbuilder so you won't have to add another module.

Reply:
Ok I'll do it

Reply:
... but we have to copy the method, right? Otherwise it's a breaking change. What do you prefer?

Reply:
Hmm the method depends on a couple of other methods in the class. Copy them all?

Reply:
done